dnf.spec

```diff
@@ -81,7 +81,7 @@
 It supports RPMs, modules and comps groups & environments.
 
 Name:           dnf
-Version:        4.2.23
+Version:        4.2.24
```

Comment:
Same here with `MINOR` vs `MICRO`, this should be a bump of `MINOR`, as the change is a feature / behaviour change, not just a bugfix. Unless we actually have some different policy for `dnf` versioning?

Reply:
Fixed.